kiwi/xml_state.py

```diff
@@ -470,6 +470,18 @@ def get_build_type_vmdvd_section(self):
             if vmdvd_sections:
                 return vmdvd_sections[0]
 
+    def get_build_type_vmconfig_entries(self):
```

Comment:
it does that the auto generated code from xml_parse::get_vmconfig_entry() returns an empty list. This is also proven by the test in test_get_build_type_vmconfig_entries